release_windows.bat

```diff
@@ -143,13 +138,14 @@ echo ***************************************************************************
 echo ****** cairosvg workaround, needs to be below 2.6.0 (isn't compatible) ******
 echo *****************************************************************************
 rem # 2023-01-30
-rem cairosvg 2.6.0 have problems with locale, so we will stay to 2.5.2 as workaround for now
+rem cairosvg >=2.6.0 generate a crash problem with locale in module projectlayer.py (Projector), 
+rem so we will stay to 2.5.2 as workaround for now
 pip uninstall cairosvg -y
```

Comment:
Is this uninstall+install still required since we've adjusted `requirements.txt`?